Assets/Modules/Enemies/EnemyMovementController.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyMovementController : MonoBehaviour
+{
+    public static event Action OnShooterIsDeadEvent;
```

Comment:
This event should be in the `EnemyLifeController` because it's not related to movement, but to life.

Reply:
The logic related to this event should also move to the other script.

Reply:
Done